packages/kbn-optimizer/src/optimizer/optimizer_config.ts

```diff
@@ -219,7 +219,7 @@ export class OptimizerConfig {
             new Bundle({
               type: 'entry',
               id: 'core',
-              publicDirNames: ['public', 'public/utils'],
+              publicDirNames: ['public'],
```

Comment:
Thanks Tiago for helping me finding this one, TIL that `src/core/public/utils` had a special treatment. The removal of the folder made the CI explode in a very non-explicit way.

Reply:
It was a publicDir, it has the same effect as defining `extraPublicDirs` to a plugin's definition. Not sure when or why we did that, but... yea.